conda/recipes/cucim/meta.yaml

```diff
@@ -38,21 +38,30 @@ requirements:
   build:
     - {{ compiler("c") }}
     - {{ compiler("cxx") }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} ={{ cuda_version }}
+    {% else %}
+    - {{ compiler('cuda') }}
+    {% endif %}
+    - cuda-version ={{ cuda_version }}
     - cmake >=3.23.1,!=3.25.0
     - make
     - ninja
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
     - click
-    - cudatoolkit ={{ cuda_version }}
+    - cuda-version ={{ cuda_version }}
     - cupy >=12.0.0
     - libcucim ={{ version }}
     - numpy 1.21
     - python
     - scikit-image >=0.19.0,<0.22.0a0
     - scipy
   run:
-    - {{ pin_compatible('cudatoolkit', max_pin='x', min_pin='x') }}
+    - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
+    {% if cuda_major == "11" %}
+    - cudatoolkit
```

Comment:
It seems odd that there are no corresponding runtime dependencies. Is everything that we need pulled via run exports for the CUDA 12 packages? Can we rely on libcucim to pull in those dependencies?

Reply:
Yeah we may be able to drop `cudatoolkit` here. Much of cuCIM just relies on CuPy either via function calls or kernels it has CuPy compile at runtime. So think depending on CuPy (and CuPy specifying its own requirements is sufficient)

Reply:
Done in commit ( https://github.com/rapidsai/cucim/pull/572/commits/229af54e820f344aa5f5cdc4768f53123fb66cee )